components/Layout.js

```diff
@@ -26,20 +30,182 @@ export default class Layout extends React.Component {
             content="width=device-width, initial-scale=1.0"
           />
         </Head>
-
         {/* Make sure the heigh of the body element is the entire view port */}
         <style global jsx>{`
-          @import url('https://fonts.googleapis.com/css?family=Roboto:400,600');
-          body {
-            font-family: 'Roboto';
-            min-height: 100vh;
-            position: relative;
-            margin: 0;
+          @import url('https://fonts.googleapis.com/css?family=Lora:400,400i,700|Montserrat:400,700');
```

Comment:
Can remove this once you switch the stylesheet URL.

Reply:
👍